Assets/LeapMotion+OVR/Scripts/LeapCameraAlignment.cs

```diff
@@ -3,40 +3,50 @@
 using System.Collections.Generic;
 using Leap;
 
+/// <summary>
+/// Implements spatial alignment of cameras and synchronization with images
+/// </summary>
 public class LeapCameraAlignment : MonoBehaviour {
-  public static bool VerboseDebuging = false;
+  [SerializeField]
+  protected LeapImageRetriever imageRetriever;
+  [SerializeField]
+  protected HandController handController;
 
-  [HideInInspector]//[Range(0,1)]
+  [Header("Alignment Targets")]
+
+  [SerializeField]
+  protected Transform leftCamera;
+  [SerializeField]
+  protected Transform rightCamera;
+  [SerializeField]
+  protected Transform centerCamera;
+  public List<LeapImageBasedMaterial> warpedImages;
```

Comment:
This list shouldn't be serialized, since LeapImageBasedMaterials already add themselves to this list automatically, so there is not a case where a user would need to assign elements to this list in the inspector.